test-framework/junit5/src/main/java/io/quarkus/test/junit/NativeTestExtension.java

```diff
@@ -71,6 +75,23 @@ public void beforeAll(ExtensionContext extensionContext) throws Exception {
         }
     }
 
+    private void ensureNoInjectAnnotationIsUsed(Class<?> testClass) {
+        Class<?> current = testClass;
+        while (current.getSuperclass() != null) {
+            for (Field field : current.getDeclaredFields()) {
+                Inject injectAnnotation = field.getAnnotation(Inject.class);
+                if (injectAnnotation != null) {
+                    throw new JUnitException(
+                            "@Inject is not supported in QuarkusTest tests. Offending field is "
```

Comment:
`in QuarkusTest tests` -> `in NativeImageTest tests`?

Reply:
Thanks!